app/controllers/my_tasks_controller.rb

```diff
@@ -41,11 +41,12 @@ def destroy
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_mytask
-      @mytask = Mytask.find(params[:id])
+      @mytask = MyTask.find(params[:id])
     end
 
     # Only allow a trusted parameter "white list" through.
-    def mytask_params
-      params.fetch(:mytask, {})
+    def my_task_params
+      params.require(:newMyTaskObject).permit(:task_id,:user_id,:due_date,:status,:frequency,:notes)
```

Comment:
Traditionally in Rails, we have our parameters snake-cased.  So this could be `require(:new_my_task_object)...`

What you have will work.  There is no magic in how you case the parameters.  I'm approving the PR but leaving this up to the team.  If you want to conform to snake case great, I recommend it.  Otherwise, make sure you are consistent.  Having a mix of the two would be a code smell.